src/UserInterface/CarMap.py

```diff
@@ -33,6 +33,47 @@ def __init__(self, environment_manager: EnvironmentManager, sio: AsyncServer):
 
         self._sio: AsyncServer = sio
 
+        try:
+            self.__proximity_range: int = int(self.config_handler.get_configuration()["vehicle_takeover"]
+                                                     ["proximity_timer"]["range"])
+        except KeyError:
+            logger.warning("No valid value for driver: proximity_timer/range in config_file. Using default "
+                           "value of 200")
+            self.__proximity_range = 200
+
+        try:
+            self.__can_virtual_cars_hack: boolean = bool(self.config_handler.get_configuration()["vehicle_takeover"]
+                                                     ["cars_that_can_hack"]["virtuaL"])
+        except KeyError:
+            logger.warning("No valid value for driver: cars_that_can_hack/virtuaL in config_file. Using default "
+                           "value of true")
+            self.__can_virtual_cars_hack = True
+
+        try:
+            self.__can_physical_cars_hack: boolean = bool(self.config_handler.get_configuration()["vehicle_takeover"]
+                                                     ["cars_that_can_hack"]["physical"])
+        except KeyError:
+            logger.warning("No valid value for driver: cars_that_can_hack/physical in config_file. Using default "
+                           "value of false")
+            self.__can_physical_cars_hack = False
+
+        try:
+            self.__can_virtual_cars_be_hacked: boolean = bool(self.config_handler.get_configuration()["vehicle_takeover"]
+                                                     ["hackable_cars"]["virtuaL"])
+
+        except KeyError:
+            logger.warning("No valid value for driver: hackable_cars/virtuaL in config_file. Using default "
+                           "value of False")
+            self.__can_virtual_cars_be_hacked = False
+
+        try:
+            self.__can_physical_cars_be_hacked: boolean = bool(self.config_handler.get_configuration()["vehicle_takeover"]
+                                                     ["hackable_cars"]["physical"])
+        except KeyError:
+            logger.warning("No valid value for driver: hackable_cars/physical in config_file. Using default "
+                           "value of True")
+            self.__can_physical_cars_be_hacked = True
+
```

Comment:
The app needs to be restarted for config changes to take effect. It would be better, if the config value are read when they are needed.